js/ui/viewSelector.js

```diff
@@ -416,6 +476,8 @@ const ViewsClone = new Lang.Class({
                                                           enabled: false });
         iconGridClone.add_effect(this._saturation);
 
+        if (discoveryFeed !== null)
+            this.add_child(discoveryFeed);
```

Comment:
Instead of checking whether discoveryFeed is null here, you can probably just add the child in the parent if condition